src/gui/Preferences.h

```diff
@@ -70,7 +70,7 @@ static json defaultPrefs = {
         {"fire", "E"},
         {"boost", "Left Shift"},
         {"verticalMotion", "Left Ctrl"},
-        {"viewRange", "T"},
+        {"viewRange", ""},
```

Comment:
didn't mean to include this here, but this is removed completely in another pull request: https://github.com/avaraline/Avara/pull/138